doc/user-guides/dns/core-dns.md

```diff
@@ -36,167 +39,129 @@ From the root of the kuadrant-operator repo execute:
 
-To setup the DNSProvider secrets, you need to know the external IPAddress of each CoreDNS instance. This is to allow the DNS Operator to query each nameserver for its records to form a full record set for a given dns name.
+### A Kubernetes cluster
 
+Next you need to set the nameservers in the provider secret, as follows:
+```
+kubectl create secret generic core-dns --namespace=<COREDNS NAMESPACE> --type=kuadrant.io/coredns --from-literal=NAMESERVERS="<NAMESERVER>:53,<NAMESERVER>:53,..." --from-literal=ZONES="<DOMAIN IN COREDNS CONFIGURATION>,<DOMAIN IN COREDNS CONFIGURATION>,..."
+```
+
+### Local setup
+In the local kind cluster, these commands will export the coreDNS IPs as envvars:
 ```
 export coredns1IP=$(kubectl get service kuadrant-coredns-1 -n kuadrant-coredns-1 -o=jsonpath='{.status.loadBalancer.ingress[0].ip}')
 export coredns2IP=$(kubectl get service kuadrant-coredns-2 -n kuadrant-coredns-2 -o=jsonpath='{.status.loadBalancer.ingress[0].ip}')
 ```
 
-Next you need to set the nameservers in the provider secret:
-
+These will set up the 2 CoreDNS provider secrets:
 ```
-
 kubectl create secret generic core-dns --namespace=kuadrant-coredns-1 --type=kuadrant.io/coredns --from-literal=NAMESERVERS="${coredns1IP}:53,${coredns2IP}:53" --from-literal=ZONES="k.example.com"
 
 kubectl create secret generic core-dns --namespace=kuadrant-coredns-2 --type=kuadrant.io/coredns --from-literal=NAMESERVERS="${coredns2IP}:53,${coredns1IP}:53" --from-literal=ZONES="k.example.com"
 ```
 
-### Setup the DNSPolicies
-
-Now that you have your gateways and providers in place, you can move on to setup the DNSPolicies. Again, for convenience, there is example for this guide.
-
-```
-kubectl apply -f examples/coredns/dnspolicies.yaml
-```
-
->Note:  2 DNSPolicies are created. In a real world scenario each of these gateways and policies would likely be on a different cluster. In the 2 DNSPolicies the main difference is the one targeting the NA gateway has a GEO set of NA while the one targeting the EU gateway has a GEO set of EU.
-
+## Enable CoreDNS in the DNS Operator
 
-### Validate and test the setup
-
-You can verify that everything is as expected by checking the DNSPolicy status. It is likely to take a minute or so before the status is marked as enforced but you should see `enforced true` for each policy.
+The CoreDNS provider is not enabled by default in the DNS Operator, as this is only in developer preview currently, to enable it in a normal Kuadrant installation, run the following command:
 
 ```
-kubectl get dnspolicy -A -o=wide
+kubectl patch deployment dns-operator-controller-manager  --type='json' -p='[{"op": "replace", "path": "/spec/template/spec/containers/0/args", "value": ["--metrics-bind-address=:8080", "--leader-elect","--provider=aws,google,azure,coredns"]}, {"op": "replace", "path": "/spec/template/spec/containers/0/image", "value": "quay.io/kuadrant/dns-operator:v0.14.0"}]'  -n kuadrant-system
```

Comment:
Changing the DNS Operator image should not be required once the next release of kuadrant is out.

Reply:
> What image should this be referencing? On main having it use main made sense, but if we do this do we need to update it on all releases?

Thinking on it, do we need to set the image at all? Current release docs won't mention these docs anywhere, and running off main will just use main of dns operator. As we keep releasing, as you say, the dns operator image just needs to be left alone.

Anyway, if we leave this we should ensure it's removed after we do the release as you say.

Reply:
I had to change this, for the dns operator to work, maybe as I installed from it helmcharts? It was setting it to v0.13.0 and erroring over the unknown provider "coredns".

Reply:
OK, leave it i guess. Never used the helm chart, so no idea what it does tbh.